src/ontology/mondo.Makefile

```diff
@@ -759,6 +759,22 @@ migrate-%: tmp/mondo-edit-%.ttl
 
 migrate: migrate-omim
 
+GARD_MAPPINGS=https://docs.google.com/spreadsheets/d/e/2PACX-1vR3AUD0NRuKsFg6dNxSGSc-zabYn57udSmajScz2_puXOEZbAmEyjyyM9508OnjoJ9Ir21-BnPAJ-KX/pub?gid=1648437389&single=true&output=tsv
+
+.PHONY: update-gard-mappings
+update-gard-mappings:
```

Comment:
Not saying this is needed, but for the workflows in the `mondo` repository, do we have the same goal of adding documentation e.g. `--help` or a `workflows.md`?

Reply:
I need to really do that, but I will do it all at once.